### DISCUSSION TITLE
Schema of tables

### DISCUSSION THREAD

**Discussion by Suchit027** [Suchit027/DB---Project#3] [Ideas]
This is for us to keep track of all attributes we come up with for table schemas. So write your insights in this; later we can combine all to create a proper work flow.

**Comment by Suchit027** [comment]
So we have 3 main tables - patient, hospital, liver. Attributes suggested by me -
Patient - id, medical details, hospital id, alcoholism degree, age
Hospital - id, coordinates
Liver - medical details
Additional attributes if we have time - 
Hospital - doctors available for surgery, beds available
Liver - size, time after retrieval

**Reply by KING-258** [reply]
Add another table of reports with a foreign key constraint patient id and remove medical details from the patient table
